plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -605,12 +610,18 @@ public void createTable(Table table, PrincipalPrivileges principalPrivileges)
     public void dropTable(String databaseName, String tableName, boolean deleteData)
     {
         Table table = getExistingTable(databaseName, tableName);
+        RetryPolicy<?> concurrentModificationExceptionRetryPolicy = RetryPolicy.builder()
+                .handleIf(throwable -> Throwables.getRootCause(throwable) instanceof ConcurrentModificationException)
```

Comment:
Usually you'd just say `.handle(ConcurrentModificationException.class)`, unless the exception is wrapped in something else?

Reply:
it is wrapped by a TrinoException